src/FakeItEasy.dotnet/project.json

```diff
@@ -18,8 +18,6 @@
   },
 
   "frameworks": {
-    "netstandard1.6": {
-      "imports": [ "dotnet5.4" ]
-    }
+    "netstandard1.6": { }

```

Comment:
> Does that imply that dotnet5.4 is a subset of netstandard1.6?

Not exactly. It just means that you know something NuGet doesn't: the fact that `dotnet5.4` is compatible with `netstandard1.6`. If you're wrong, it will probably fail at runtime. So yes, it's dangerous; it exists only to enable using libraries which don't explicitly target a `netstandard` platform, but are actually compatible.